doc/howtos/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,184 @@
+Using Ubuntu as the Service OS with the ACRN hypervisor
+
+
+Based on the ACRN open source project and getting started guide. This documentation is to guide people how to boot Ubuntu system as the 
+Service OS with the ACRN hypervisor. By default, in the getting started guide documentation of the ACRN open source project, Clear Linux
+is being used as the Service OS and User OS on top of the ACRN hypervisor. 
+Here we are trying to boot Ubuntu system as the Service OS and launch Clear Linux as User OS with the ACRN hypervisor on official 
+supported NUC, playing steps as below:
+
+Step #1,  Installing a Ubuntu system on the NUC with standard installing steps. 
+For example with Ubuntu 16.04.4 LTS distribution.
+Download it here: https://www.ubuntu.com/download/desktop
+Burn Ubuntu 16.04.4 image to a USB key
+Install Ubuntu system with the USB key 
+After complete installation and rebooting, you could boot Ubuntu from grub menu.
+
+Step #2,  setup proxy with you prefered ways, command line tool or system settings tool.  If you don’t need to setup proxy, you could 
```

Comment:
I would leave this out completely. Setting up the proxy is what I'd consider part of the usual installation steps. We could at most have a note that invites them to make sure they have internet access as we'll need to download some stuff later.